scripts/source-maintenance.sh

```diff
@@ -68,6 +76,10 @@ if test "${ASVER}" != "2.04" ; then
 else
 	echo "Found astyle ${ASVER}. Formatting..."
 fi
+if test "${SkipAstyleVersionCheck}" = "yes" && test -z "${ASVER}" ; then
```

Comment:
Please move this (adjusted) check into the `if` statement where we discover the problem. It would clarify the logic, especially because the official code is abusing ASVER for enabling or disabling formatting.

Reply:
changed logic. Resolving comment.